lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -701,7 +701,7 @@ def __init__(
             opener: specifies an opener for unix socket, if unspecified a default is used
             base_url: base url for making requests to the snap client. Defaults to
                 http://localhost/v2/
-            timeout: timeout in seconds to use when making requests to the API. Default is 5.0s.
+            timeout: timeout in seconds to use when making requests to the API. Default is 30.0s.
```

Comment:
It's a pity the [online lib docs](https://charmhub.io/operator-libs-linux/libraries/snap#snapclient) don't show these defaults in the `SnapClient.__init__` signature so that we have to repeat them here, but oh well.